boards/arm/olimex_stm32_p103/olimex_stm32_p103_defconfig

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F1X=y
+CONFIG_SOC_STM32F103XB=y
+# 64MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=64000000
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Clock configuration for Cube Clock control driver
+CONFIG_CLOCK_STM32_HSE_CLOCK=8000000
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# use HSE as PLL input
+CONFIG_CLOCK_STM32_PLL_SRC_HSE=y
+# produce 64MHz clock at PLL output
+CONFIG_CLOCK_STM32_PLL_MULTIPLIER=8
+CONFIG_CLOCK_STM32_AHB_PRESCALER=1
+CONFIG_CLOCK_STM32_APB1_PRESCALER=4
+CONFIG_CLOCK_STM32_APB2_PRESCALER=2
```

Comment:
I use the following configuration to run everything at the highest allowed clock for this MCU:
```
CONFIG_CLOCK_STM32_PLL_MULTIPLIER=9
CONFIG_CLOCK_STM32_AHB_PRESCALER=1
CONFIG_CLOCK_STM32_APB1_PRESCALER=2
CONFIG_CLOCK_STM32_APB2_PRESCALER=1
```

Reply:
Makes sense. These settings are the same as for [nucleo_f103rb](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nucleo_f103rb/nucleo_f103rb_defconfig).